src/app/home/pages/ClustersPage/components/AddEditClusterModal/__tests__/AddEditClusterModal.test.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from '../../../../../../../reducers';
+
+import AddEditClusterModal from '../index';
+
+const store = createStore(rootReducer, {});
+
+describe('<AddEditClusterModal />', () => {
+  it('adds a cluster with valid values', () => {
```

Comment:
Small nit-pick, since this is just testing form validation and not actually adding a cluster, maybe we should call it something like "enables add cluster button when values are valid"?